tests/api_connexion/schemas/test_task_instance_schema.py

```diff
@@ -199,7 +199,6 @@ def test_success(self):
     @parameterized.expand(
         [
             ({"task_id": None},),
-            ({"include_future": "True"},),
```

Comment:
We shouldn't delete this, rather we should use marshmallow none truthy string. Please add it back and replace "True" with "mystring"

Reply:
To be quite honest, I'm not sure what exactly are we going to test by this thing? I suppose that we shouldn't test the Marshmallow functionality (i.e. correctness of `fields.Boolean` work) as it's not our area of responsibility. However, if we would like to do so, why not all the fields are tested on loading any wrong type with failure? What's specific about this `include_future` here?

Reply:
The OpenAPI spec validates these things for us. We shouldn't even be writing many of these Marshmallow schemas but since it's part of our code now and we are writing it, we should write it correctly.
And there's nothing specific about the `include_future`, it's just like other fields. Since we have tests for the other fields, let's also have a test for it

Reply:
Well, no problem, I returned it back.